coding-test/sangjun/binary-serach/떡볶이떡만들기.js

```diff
@@ -0,0 +1,24 @@
+const fs = require('fs');
+let input = fs.readFileSync('/dev/stdin').toString().split('\n');
+
+const [n, m]= input[0].split(' ');
+const array = input[1].split(' ').map(Number).sort((a, b) => a - b);
+
+let start = 0;
+let end = Math.max.apply(0, array);
+let result = 0;
+
+while (start <= end) {
+    let total = 0;
+    const mid = parseInt((start + end) / 2);
```

Comment:
가능하면 상수를 최상단에 배치.
또한, for 문에서 total을 사용하므로 for 위에 total을 선언하는 것이 맞음.

```suggestion
    const mid = parseInt((start + end) / 2);
    let total = 0;
```